kotsadm/pkg/snapshot/backup.go

```diff
@@ -168,43 +207,43 @@ func createAdminConsoleBackup(ctx context.Context, isScheduled bool) error {
 	veleroBackup := &velerov1.Backup{
 		ObjectMeta: metav1.ObjectMeta{
 			Name:         "",
-			GenerateName: "kotsadm-",
+			GenerateName: "instance-",
```

Comment:
I'm assuming this naming was discussed with @markpundsack

Reply:
Hmmm, It was not actually. @markpundsack Any objections/suggestions?

Reply:
This is for the prefix of the backup name? I guess that works. We've been talking about instance vs app snapshots. We've also talked about full vs partial backups. It's no longer only Kots Admin Console, so it makes sense not to call it that. But another option could be simply `kots-`. Instance is such an ambiguous term. But, I don't have strong objections.

Reply:
BTW, do we expect that if someone has backups of different Kots instances into the same S3 bucket, they use different paths? When showing a list of backups, do we show all backups in that bucket/path? Is the name meaningful at all for identifying Kots backups?

Reply:
Or (and I hope this isn't the case) does there need to be a record of each backup in the cluster?

Reply:
Currently, the name is not meaningful at all to identify different instances of kots backups. However, we do have annotations added to the Backup object to help identify that.

Reply:
Might be a good idea to have those as labels instead of annotations